OOVPA.h

```diff
@@ -73,7 +73,13 @@ typedef struct _LOVP
     // large offset. Value can be safely cast to 8-bit for OOVPA, but must
     // remain 16-bit for XRef entries.
     unsigned short Offset;
-    unsigned short Value;
+    union {
+        unsigned short sValue;
+        struct {
+            unsigned char unused;
+            unsigned char cValue;
```

Comment:
Weird prefix style. You'd normally see `b` for byte (`unsigned char`) in MS style code.